src/components/Authentication/ResetPassword.jsx

```diff
@@ -1,54 +1,116 @@
+import {
+  Container,
+  Flex,
+  Heading,
+  Spacer,
+  Text,
+  Box,
+  InputGroup,
+  Input,
+  InputRightElement,
+  Button,
+  Center,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Center' is defined but never used.